proxy/src/avatar.rs

```diff
@@ -0,0 +1,201 @@
+//! Org and user avatar generation.
+
+/// Emoji whitelist.
+const EMOJIS: &[&str] = &[
+    "⌚️", "📱", "📲", "💻", "⌨️", "💽", "💾", "💿", "📀", "📼", "📷", "📸", "📹", "🎥", "🎞", "📞",
+    "☎️", "📟", "📠", "📺", "📻", "⏰", "🕰", "⌛️", "⏳", "📡", "🔋", "🔌", "💡", "🔦", "💸", "💵",
+    "💴", "💶", "💷", "💰", "💳", "🧾", "💎", "⚖️", "🔧", "🔨", "", "🔩", "⚙️", "⛓", "🔫", "💣", "🔪",
+    "🗡", "⚔️", "🚬", "⚰️", "⚱️", "🏺", "🧭", "🧱", "🔮", "🧿", "🧸", "📿", "💈", "⚗️", "🔭", "🧰", "🧲", "🧪",
+    "🧫", "🧬", "🧯", "🔬", "🕳", "💊", "💉", "🚽", "🚰", "🚿", "🛁", "🛀", "🛀🏻", "🛀🏼", "🛀🏽",
+    "🛀🏾", "🛀🏿", "🧴", "🧵", "🧶", "🧷", "🧹", "🧺", "🧻", "🧼", "🧽", "🛎", "🔑", "🗝", "🚪", "🛍", "🧳",
+    "🛒", "🎁", "🎈", "🎏", "🎀", "🎊", "🎉", "🧨", "🎎", "🏮", "🎐", "🧧", "✉️", "📩", "📨", "📧",
+    "💌", "📥", "📤", "📦", "📪", "📫", "📬", "📭", "📮", "📯", "📜", "📃", "📄", "📑", "📊", "📈",
+    "📉", "📆", "📅", "📇", "📋", "📁", "📂", "🗂", "🗞", "📰", "📓", "📔", "📒", "📕", "📗", "📘",
+    "📙", "📚", "📖", "🔖", "🔗", "📎", "📐", "📏", "📌", "📍", "✂️", "🖊", "🖋", "✒️", "🖌", "📝", "✏️",
+    "🔍", "🔎", "🔏", "🔐",
+];
```

Comment:
:information_source: We can't use `char` here, because some emojis use more than one unicode codepoint.

Reply:
Maybe we can capture that in a inline comment.